src/app/routes/public-forms.server.routes.js

```diff
@@ -68,23 +69,36 @@ module.exports = function (app) {
   /**
    * @typedef Feedback
    * @property {number} rating.required - the user's rating of the form
-   * @property {string} comment.required - any comments the user might have
+   * @property {string} comment - any comments the user might have
    */
 
   /**
    * Send feedback for a public form
-   * @route POST /{formId}/feedback
+   * @route POST /:formId/feedback
    * @group forms - endpoints to serve forms
    * @param {string} formId.path.required - the form id
    * @param {Feedback.model} feedback.body.required - the user's feedback
    * @consumes application/json
    * @produces application/json
-   * @returns {string} 400 - form feedback was malformed and hence cannot be saved
-   * @returns {string} 200 - form feedback was saved
+   * @returns 200 if feedback was successfully saved
+   * @returns 400 if form feedback was malformed and hence cannot be saved
+   * @returns 404 if form with formId does not exist or is private
+   * @returns 410 if form has been archived
+   * @returns 500 if database error occurs
    */
-  app
-    .route('/:formId([a-fA-F0-9]{24})/feedback')
-    .post(forms.formById, publicForms.isFormPublic, publicForms.submitFeedback)
+  app.route('/:formId([a-fA-F0-9]{24})/feedback').post(
+    celebrate({
+      [Segments.BODY]: Joi.object()
+        .keys({
+          rating: Joi.number().min(1).max(5).cast('string').required(),
+          comment: Joi.string().allow('').required(),
+        })
+        // Allow other keys for backwards compability as frontend might put
+        // extra keys in the body.
+        .unknown(true),
```

Comment:
is this because of `isPreview`? if so might it be a better idea to add `isPreview` as a key for now, remove it in the frontend, then remove it from the backend a week later?

Reply:
As discussed, keep `unknown` in and remove in the future when frontend stops sending `isPreview` instead of making the API "looser"